### ISSUE TITLE
Add bold formatting to usernames in message strings on both Telegram and IRC

### ISSUE THREAD

**Issue by jwflory** [RITlug/teleirc#19]
# Summary

When using the bridge bot, it would be helpful if usernames were marked up with bold text, to help separate them from the message body on IRC and Telegram.
# Analysis

In #8, there was a lot of discussion about how this should be implemented, especially in terms of using pre-existing libraries for markdown. Before completing or coming up with a solution to close this issue, please read through the comments and discussion in #8.

**Comment by jflory7** [comment]
@repkam09 or @thenaterhood, I know we had a PR that was almost there for this, but do you think it's something we might be able to implement ourselves? This would be a nice feature to make it more clear that the teleirc bot has multiple users behind (e.g. people who reply to messages pinging `teleirc`).

**Comment by repkam09** [comment]
Is this on the IRC side or the Telegram side, or both? How do you force bold characters in the terminal side? If its as easy as sticking a *name* in there, or an escape character maybe it could be done earily.

On the Telegram side, there is a formatted text mode that a bot can use instead of just plain text. I looked up how to do it last time this issue was suggested. I would have to read up again on the specifics.

**Comment by abkahrs** [comment]
@repkam09 @jflory7 for bolding on the irc side when a message comes through from telegram it should work to put the username string through the bold() method before the full message is assembled and sent through to irc. Not sure going the other way.

**Comment by ImTheDeveloper** [comment]
You can use markdown to use bold when sending messages in bold over telegram. I believe using *** works

**Comment by jflory7** [comment]
@ImTheDeveloper If you'd like to submit a PR for this, that would be awesome, if it's something simple! You'd probably be faster to implement it than me, anyways. Unless @repkam09 or @xforever1313 wanted to take a crack at this.

**Comment by dennisroczek** [comment]
please also add that the IRC comment 

> \me blabla 

gets "highlighted" in another color or so. ;-) at least recognizable in telegram

**Comment by jflory7** [comment]
If working on editing the message strings between the bridge, it might be helpful to also take a look at #44. It could be possible to do both of these in one go.

**Comment by jwflory** [comment]
_Discussed during 2019-04-06 developer meeting._

---

@nic-hartley is picking this issue up to work on. If it is as easy as we suspected, he hopes to have a pull request up by Saturday, April 13.

In the meeting, we clarified this is _NOT_ bold highlighting of usernames mentioned in a message, but it is highlighting the message sender's username in their relayed message to the other side. Consider this example:

```
IRC message: <jwf> Fuego is the best!
Relayed to Telegram: <**jwf**> Fuego is the best!
```

And vice versa.
